js/kasten.js

```diff
@@ -74,6 +74,23 @@ function updateBoxClass(schoolClass) {
   boxContainer.innerHTML += box;
 }
 
+var infoContainer = document.getElementById('info-container');
+function updateInfoDateBox(date) {
+  var box = '<div class="info-kasten info-date">'
+            + date +
+            '</div>';
+
+  infoContainer.innerHTML += box;
+}
+
+function updateInfoBox(info) {
+  var box = '<div class="info-kasten">'
```

Comment:
[Source](https://github.com/CoderDojoPotsdam/regex-tutorial/blob/master/js/playfield.js#L17)
```
  var element = document.createElement('div');
  element.innerText = date;
  element.classList.add("info-kasten");
  element.classList.add("info-date");
  infoContainer.appendChild(element);
```
This creates the element all in JavaScript and does not make all elements to strings and then, arses the string again to make elements.

This approach is `O(1)` whereas your approach is `O(n)`. ([O-Notation](https://de.wikipedia.org/wiki/Landau-Symbole) für Dauer von Algorithmen)